snapcraft.yaml

```diff
@@ -286,3 +290,33 @@ parts:
       - libglib2.0-dev
     stage-packages:
       - libglib2.0-bin
+  ubuntu-app-platform:
```

Comment:
This stenza isn't necessary. It was for backward compatibility reason with the new desktop- syntax. You can remove it as we never published that new part with the desktop/ syntax.